client/src/commons/LoadingSpinner/LoadingSpinner.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { CircularProgress } from '@material-ui/core';
+
+import StoreStateType from 'redux/storeStateType';
+
+import useStyle from './LoadingSpinnerStyles';
+
+const LoadingSpinner: React.FC = () => {
+
+    const isLoading = useSelector<StoreStateType, boolean>(state => state.isLoading);
+
+    const classes = useStyle();
+
+    return (
+        <>
+        {
+            isLoading &&
+                <div className={classes.container}>
+                    <CircularProgress size='20vh'/>
```

Comment:
Why inline css?

Reply:
it is not inline css, its the property in the component that decides the size of the spinner, the only way to control it

Reply:
Ok sorry :(